pulsar-broker/src/test/java/org/apache/pulsar/broker/service/RackAwareTest.java

```diff
@@ -79,7 +84,7 @@ protected void setup() throws Exception {
 
     }
 
-    @AfterClass(alwaysRun = true)
```

Comment:
is this necessary ?

Reply:
there's some oddities when using the TestNG After/Before annotations in tests. There's some cases where the method doesn't get called at all. 

@Technoboy- I made some observations that super class After/Before annotations will only get called if they are public methods. It must be some kind of TestNG bug. Please change "protected void cleanup" to "public void cleanup" both in this class and also in the base class which is BkEnsemblesTestBase.

Reply:
Please don't remove this annotation.